### ISSUE TITLE
Follow the system's Light/Dark setting

### ISSUE THREAD

**Issue by daveliao** [OptimiliaStudios/AquileReader#5]
Currently the app retains its custom colors regardless of my system's Light/Dark setting (in Colors).

This reference might help - https://docs.microsoft.com/en-us/windows-hardware/customize/desktop/set-dark-mode

**Comment by OptimiliaStudios** [comment]
Hello,
Since the app allows users to set their own color schemes (with dark/light mode) for both Reader and rest of the app, what do you suggest I switch to which color scheme when the system color settings' change?

Will it make sense if I have a system color theme option, which uses the system's accent color and mode?

**Comment by johnwang16** [comment]
I think it would make the most sense to have the user able to specify which color scheme to use in light mode and which to use in dark mode. That would save you from having to figure out what colors to use.

**Comment by daveliao** [comment]
@OptimiliaStudios - I love @johnwang16 's solution - that saves you the effort of trying to figure that out yourself.

Also, I just realized I know John in real life! **HI JOHN!**

**Comment by v0iden** [comment]
I agree with @johnwang16 's proposal. This is a common way of doing it, example from Visual Studio Code: 
![image](https://user-images.githubusercontent.com/31285047/146243361-4c349a95-013d-4a57-ae4d-0f9f793c198c.png)

**Comment by loyukfai** [comment]
Yes, please do it, thanks!

**Comment by loyukfai** [comment]
Is this repository being monitored actually?

**Comment by loyukfai** [comment]
Hi anyone?

**Comment by loyukfai** [comment]
Hi anyone......

**Comment by loyukfai** [comment]
Hello~

**Comment by loyukfai** [comment]
Summer greetings. Any news? : )

**Comment by loyukfai-new** [comment]
Hi I have this problem too...

**Comment by loyukfai** [comment]
Hi~

**Comment by loyukfai** [comment]
Happy New Year! Any news? 😊

**Comment by OptimiliaStudios** [comment]
The Android version of the app has this support, we would try and add this support on Windows as well.
Right now we are focused on improving the PDF annotation experience of the app.

**Comment by loyukfai** [comment]
> The Android version of the app has this support, we would try and add this support on Windows as well. Right now we are focused on improving the PDF annotation experience of the app.

Aye aye thanks!

**Comment by loyukfai** [comment]
Hi any news?

**Comment by loyukfai** [comment]
Hi any news?

**Comment by loyukfai** [comment]
Hi any news?
